### DISCUSSION TITLE
High GPU utilization after loading snapshots

### DISCUSSION THREAD

**Discussion by Doraemon167** [NVlabs/instant-ngp#794] [Q&A] [answered]
Dear author, 
           the GPU utilization rate is high after loading the snapshot. I want to know how to reduce the GPU utilization rate when there is no keyboard and mouse operation? Thanks.

**Comment by CCOSerika** [comment]
I have the same problem. Looking forward to a solution.

**Comment by Tom94** [comment, marked as answer]
Hi there, that's because __instant-ngp__ continually renders frames for higher and higher super-sampling.

To disable this, pull the latest commit from `master` and then set the `Max spp` GUI option to a value above zero (corresponding to the number of frames that will be rendered once the camera becomes stationary). Cheers!

**Reply by Doraemon167** [reply]
Thank you,Tom. I will try it.

**Reply by CCOSerika** [reply]
It performs perfectly. Thanks a lot!
